src/Components/AddNewComment.jsx

```diff
@@ -0,0 +1,82 @@
+import { useState, useEffect } from "react";
+import React from "react";
+// import { useUserContext } from "./UserContext";
+
+const AddNewComment = ({setComments, article_id }) => {
+
+  const [commentBody, setCommentBody] = useState("");
+//   const { user } = useUserContext();
+  const [error, setError] = useState(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+//   console.log(user, 'user');
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+}
+
+const requestOptions = {
+    method: "POST",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify({
+      body: commentBody,
+    //   username: `${user.username}`,
+    }),
+  };
+
+    useEffect(() => {
+      setIsLoading(true);
+      fetch(
```

Comment:
Ideally we should be using axios here, the functionality that axios offers will be really useful later on and generally handles things nicer, take a look at the notes on axios to help you with this. Also we should ideally be extracting our api calls out to their own api.js file in the root of the project and calling them where we want them. This keeps things tidy and allows our api calls to be configured in one place.